tests/VerbalExpressions/PHPVerbalExpressions/VerbalExpressionsTest.php

```diff
@@ -574,7 +574,7 @@ public function testReplace()
         $this->assertEquals('bazbarbaz', $regex->replace('foobarfoo', 'baz'));
     }
 
-
+    /*
```

Comment:
That would just be me making quick edits to try and get travis testing to pass :P